[2020-06-12 16:55:34,714] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [queued]>
[2020-06-12 16:55:34,745] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [queued]>
[2020-06-12 16:55:34,745] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-12 16:55:34,745] {taskinstance.py:880} INFO - Starting attempt 27 of 29
[2020-06-12 16:55:34,745] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-12 16:55:34,767] {taskinstance.py:900} INFO - Executing <Task(BashOperator): modeling> on 2020-06-11T00:00:00+00:00
[2020-06-12 16:55:34,770] {standard_task_runner.py:53} INFO - Started process 1095 to run task
[2020-06-12 16:55:34,866] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [running]> vinicius-Latitude-3480
[2020-06-12 16:55:34,891] {bash_operator.py:82} INFO - Tmp dir root location: 
 /tmp
[2020-06-12 16:55:34,892] {bash_operator.py:103} INFO - Temporary script location: /tmp/airflowtmp025kckc0/modeling61xd3if0
[2020-06-12 16:55:34,892] {bash_operator.py:115} INFO - Running command: python /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/src/modeling/run.py
[2020-06-12 16:55:34,897] {bash_operator.py:122} INFO - Output:
[2020-06-12 16:56:59,342] {bash_operator.py:126} INFO - Treinando modelo KN_Test_2
[2020-06-12 16:56:59,342] {bash_operator.py:126} INFO - Treinando modelo KN_4
[2020-06-12 16:56:59,342] {bash_operator.py:126} INFO - Treinando modelo test_LR
[2020-06-12 16:56:59,342] {bash_operator.py:126} INFO - Tabela salva em /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/data/output/model_output_91819.csv
[2020-06-12 16:56:59,342] {bash_operator.py:126} INFO - dados dos modelos salvos em /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/data/output/model_output_91819.csv
[2020-06-12 16:56:59,663] {bash_operator.py:128} INFO - Command exited with return code 0
[2020-06-12 16:56:59,686] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=dataops, task_id=modeling, execution_date=20200611T000000, start_date=20200612T195534, end_date=20200612T195659
[2020-06-12 16:57:00,247] {logging_mixin.py:112} INFO - [2020-06-12 16:57:00,247] {local_task_job.py:103} INFO - Task exited with return code 0
